core/extensions/OutOfProcCOM/ExeServer/ExeServer.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Contract\Server.Contract.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Also checked that the `ExeServer` still works when switched to net5.0 and published as single-file.